package.json

```diff
@@ -36,25 +36,26 @@
   },
   "homepage": "https://github.com/Agoric/documentation#readme",
   "dependencies": {
-    "@agoric/zoe": "beta",
-    "@endo/far": "^0.2.19",
-    "@endo/marshal": "^0.8.6",
-    "@endo/patterns": "^0.2.3",
-    "@endo/pass-style": "^0.1.6",
+    "@agoric/zoe": "^0.26.3-u13.0",
+    "@endo/far": "^1.0.1",
+    "@endo/marshal": "^1.0.1",
```

Comment:
Are these 1.0.x endo versions on purpose? The marshal version corresponding to zoe 0.26.3-u13.0 is 0.8.5.

https://github.com/Agoric/agoric-sdk/blob/agoric-upgrade-13/packages/zoe/package.json#L59

Reply:
I just used `yarn add` to grab the latest. I don't think it makes a difference here but if it does let's get that documented and tested.

Reply:
Does https://github.com/Agoric/agoric-sdk/issues/8621 count as documentation? That's my concern.
It's not directly relevant here, but I hope the way we do stuff in this documentation repo will converge with the way we recommend doing stuff in dapps.

@mhofman @kriskowal can I get your thoughts here?

Reply:
Just seeing this now (after my approval and turadg’s merge) but I don’t have much to offer. My intuition is that users will expect installing the latest of everything to be in a consistent state and if it works, it works. If it doesn’t work, then there are surprising remediations, but that’s par for the _gestures broadly_ the emergent chaos of intractable priors.

Reply:
I chatted yesterday with @mhofman @turadg and @michaelfig and I think the conclusion is that we’re going to start having the Agoric SDK release process add `agoric-` prefix tags to the corresponding `@endo/*` releases so that one can `yarn install @endo/x@tag @agoric/y@tag` and expect a coæval snapshot.